app/src/main/java/org/oppia/android/app/story/StoryFragmentPresenter.kt

```diff
@@ -76,7 +76,7 @@ class StoryFragmentPresenter @Inject constructor(
       (activity as StoryActivity).finish()
     }
 
-    binding.storyToolbar.setOnClickListener {
+    binding.storyToolbarTitle.setOnClickListener {
```

Comment:
any test case to update for this?

Reply:
No.
<img width="1466" alt="Screenshot 2021-05-24 at 4 14 29 AM" src="https://user-images.githubusercontent.com/9396084/119278994-8af55a80-bc46-11eb-84dc-7ca85e503c46.png">